framework/math/ParallelVector/ParallelVector.h

```diff
@@ -110,27 +134,16 @@ class ParallelVector
   virtual void operator+=(const ParallelVector& y) = 0;
 
   /**Adds a vector multiplied by scalar a. Optimized for a=1.0 and -1.0*/
```

Comment:
Please add the equation to the doc. I would do something like 
```
/**Adds a scaled vector to obtain \f$ \vec{x} = \vec{x} + a \vec{y} \f$. Optimized for a = 1.0 and a = -1.0.*/
```
Alternatively, add a line to the doc and us `\f[ ... \f]`.

Reply:
Ooof. The formulas look nasty in the IDE even though they might look cool in html-docs.
![image](https://github.com/chi-tech/chi-tech/assets/25042529/cb70768a-a86c-42ee-810d-b7e2cc5253b4)

I think I prefer just having it bare bones. Then it is readable in both html and IDEs.
![image](https://github.com/chi-tech/chi-tech/assets/25042529/7cad0abc-310f-478a-9829-aaff067ad28f)

Reply:
```
/**Adds a vector multiplied by scalar a, `x = x + a*y`. Optimized for a=1.0 and
   * -1.0*/
  void PlusAY(const ParallelVector& y, double a) override;```
![image](https://github.com/chi-tech/chi-tech/assets/25042529/a8875bdf-7b93-461e-8988-0df8cc84351b)